code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,328 @@
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/nullrod/claymore/saber/red
+	name = "dark energy sword"
+	icon_state = "swordred"
+	item_state = "swordred"
+	desc = "Woefully ineffective when used on steep terrain."
+
+/obj/item/weapon/nullrod/sord
+	name = "\improper UNREAL SORD"
+	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
+	icon_state = "sord"
+	item_state = "sord"
+	slot_flags = SLOT_BELT
+	force = 4.13
+	throwforce = 1
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/scythe
+	icon_state = "scythe0"
+	item_state = "scythe0"
+	name = "reaper scythe"
+	desc = "Ask not for whom the bell tolls..."
+	w_class = 4
+	armour_penetration = 35
+	slot_flags = SLOT_BACK
+	sharp = 1
+	edge = 1
+	attack_verb = list("chopped", "sliced", "cut", "reaped")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+
+/obj/item/weapon/nullrod/scythe/vibro
+	icon_state = "hfrequency0"
+	item_state = "hfrequency1"
+	name = "high frequency blade"
+	desc = "Bad references are the DNA of the soul."
+	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
+
+/obj/item/weapon/nullrod/hammmer
+	icon_state = "hammeron"
+	item_state = "hammeron"
+	name = "relic war hammer"
+	desc = "This war hammer cost the chaplain fourty thousand space dollars."
+	slot_flags = SLOT_BELT
+	w_class = 5
+	attack_verb = list("smashed", "bashed", "hammered", "crunched")
+
+/obj/item/weapon/nullrod/chainsaw
+	name = "chainsaw hand"
+	desc = "Good? Bad? You're the guy with the chainsaw hand."
+	icon_state = "chainsaw1"
+	item_state = "mounted_chainsaw"
+	w_class = 5
+	flags = NODROP | ABSTRACT
+	sharp = 1
+	edge = 1
+	attack_verb = list("sawed", "torn", "cut", "chopped", "diced")
+	hitsound = 'sound/weapons/chainsaw.ogg'
+
+/obj/item/weapon/nullrod/clown
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "clownrender"
+	item_state = "gold_horn"
+	name = "clown dagger"
+	desc = "Used for absolutely hilarious sacrifices."
+	hitsound = 'sound/items/bikehorn.ogg'
+	sharp = 1
+	edge = 1
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/whip
+	name = "holy whip"
+	desc = "What a terrible night to be on Space Station 13."
+	icon_state = "chain"
+	item_state = "chain"
+	slot_flags = SLOT_BELT
+	attack_verb = list("whipped", "lashed")
+	hitsound = 'sound/weapons/slash.ogg'
+
+/obj/item/weapon/nullrod/whip/afterattack(atom/movable/AM, mob/user, proximity)
+	if(!proximity)
+		return
+	if(istype(AM, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = AM
+		if(is_shadow(H))
+			var/phrase = pick("Die monster! You don't belong in this world!!!", "You steal men's souls and make them your slaves!!!", "Your words are as empty as your soul!!!", "Mankind ill needs a savior such as you!!!")
+			user.say("[phrase]")
+			H.adjustBruteLoss(8) //Bonus damage
```

Comment:
we may want to remove this....or just add it as a random proc for all people hit...